data-syncs/supported-data-sync-sources/soap-1.2-web-CDC.md

```diff
@@ -0,0 +1,97 @@
+# SOAP web service (Cinchy Event Triggered)
+
+## Overview
+
+SOAP (Simple Object Access Protocol) is an XML-based protocol for accessing web
```

Comment:
I know we're gonna go over this, but this page currently only describes SOAP and we need to make it a CDC triggered SOAP which has a different flow of data altogether. Lets chat

Reply:
As per the push notes, this page was copied and requires the CDC section. It's only a duplication of the SOAP 1.2 page.